### DISCUSSION TITLE
Readonly for owner?

### DISCUSSION THREAD

**Discussion by vezaynk** [codesandbox/codesandbox-client#7067] [Q&A] [answered]
I can't find the option that was implemented for #696

Where is it?

**Comment by sannek** [comment, marked as answer]
When you have a specific sandbox open, there is a toggle `frozen` underneath the title etc. (If the sidebar isn't open, click the little box on the left to open it).

Otherwise, on your dashboard, you can right-click any sandbox and chose 'freeze sandbox' from the context menu.

![image](https://user-images.githubusercontent.com/17491062/208432450-d82cb85f-4310-4baa-824c-c9378edcfba4.png)
